backend/src/models/email.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const emailSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+  },
+});
+
+const Email = mongoose.model("Email", emailSchema);
+module.exports = Email;
```

Comment:
"import" pairs up with "export" keyword in ES6 and "require" pairs up with "module.exports". To make our app works we will have to change this line to something like "export default Email;"!